### PR TITLE
Developed header and navigation 

### LINKED ISSUE #1: Header with Section Navigation
#### Description
The task involves creating the website header based on the design provided in Figma. Additionally, implement navigation links that smoothly scroll the page to different sections upon clicking.

#### Tasks
- [x] Review the Figma design for the header component.
- [x] Create HTML/CSS structure for the header.
- [x] Implement navigation links to relevant sections of the site.
- [x] Test navigation functionality and smooth scrolling.
- [x] Ensure responsiveness and compatibility across devices.

Screen Shot 2023-12-05 at 9 46 56 PM

### PR DESCRIPTION
**Pull request by alexrdrgz**
## Changes 

- Added html file and basic structure
- Add header and navigation
- Linked style.css and script.js to index.html

closes #1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (Adrean-Cajigas) [inline index.html:6]: Very creative title. I like it.

**Review by BSchoolland — CHANGES_REQUESTED** [review]
I don't see the header, but the html looks good

File: index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (ColbyGatty) [inline index.html:4]: Where is the header?

**Review by ColbyGatty — COMMENTED** [review]
Nice template, missing header!

**Pull request closed without merging**